mhubert-pmn/Front-Url-Shortener
# [Feature] layout

## Task
Create layout

## Type of change

## Description
- Create Layout.js, layout.scss
- Put background img into src
- Displace layout directory into components


## Testing scenario
In the terminal:
- Launch and build: `docker-compose up --build
- Go to [http://localhost:3000/](http://localhost:3000/) and check if the layout appears (background image)

## Trello card
[https://trello.com/c/vC9Qdvtf](https://trello.com/c/vC9Qdvtf)
